fedimint-core/src/db/mod.rs

```diff
@@ -351,8 +351,16 @@ pub trait IDatabaseTransaction<'a>: 'a + MaybeSend {
 
     async fn raw_remove_entry(&mut self, key: &[u8]) -> Result<Option<Vec<u8>>>;
 
+    /// Returns an stream of key-value pairs with keys that start with
+    /// `key_prefix`. No particular ordering is guaranteed.
     async fn raw_find_by_prefix(&mut self, key_prefix: &[u8]) -> PrefixStream<'_>;
```

Comment:
Don't need to fix in this PR but we should probably make this return `Result<PrefixStream<'_>> so that the API is consistent.